pkg/sloop/webserver/webfiles_test.go

```diff
@@ -46,3 +49,15 @@ func writeFile(t *testing.T, fs *afero.Afero, filePath string, content string) {
 	err = fs.WriteFile(filePath, []byte(content), defaultFileMode)
 	assert.Nil(t, err)
 }
+
+func Test_FileExistsInList_True(t *testing.T) {
```

Comment:
Similarly move these to https://github.com/salesforce/sloop/blob/master/pkg/sloop/common/utilities_test.go